Mathlib/Analysis/MeanInequalitiesPow.lean

```diff
@@ -61,23 +61,6 @@ theorem pow_arith_mean_le_arith_mean_pow_of_even (w z : ι → ℝ) (hw : ∀ i
     (∑ i ∈ s, w i * z i) ^ n ≤ ∑ i ∈ s, w i * z i ^ n :=
   hn.convexOn_pow.map_sum_le hw hw' fun _ _ => Set.mem_univ _
 
-/-- Specific case of Jensen's inequality for sums of powers -/
-theorem pow_sum_div_card_le_sum_pow {f : ι → ℝ} (n : ℕ) (hf : ∀ a ∈ s, 0 ≤ f a) :
```

Comment:
Should this have a deprecation?

Reply:
I can't add it here because this file doesn't import `Algebra.Order.Chebyshev`

Reply:
Then put it in another file; I think we should at least emit warnings for users of `import Mathlib`.

bors r-

bors d+

Reply:
Sorry, I completely missed that comment